packages/desktop-client/src/components/transactions/TransactionList.jsx

```diff
@@ -161,13 +164,19 @@ export function TransactionList({
   });
 
   const onNavigateToSchedule = useCallback(scheduleId => {
-    pushModal('schedule-edit', { id: scheduleId });
+    dispatch(pushModal('schedule-edit', { id: scheduleId }));
+  });
+
+  const onNavigateToFilteredTagView = useCallback(noteTag => {
+    const filterConditions = [
+      { field: 'notes', op: 'contains', value: noteTag, type: 'string' },
```

Comment:
Should this be `op: 'is'` instead to pull only unique tag names?

Reply:
Tried experimenting with using `is`. The problem with using it is that it won't be able to support multiple tags and non-tag texts (no #) in the notes.

Reply:
Can you give specific examples of these? Eg. `notes contains ...`

Reply:
You can try adding a note with a tag and regular note e.g. `A sample #tag with a regular note`. If you click on the tag it won't match the transaction.

Reply:
Okay, I see now. So the PR isn't actually creating tags or keeping a list of created tags. It's just formatting the notes to give a pill around anything starting with #. So this is primarily a visual change not a functional addition.

Would be good (in the future) to make tags an actual saved item (new db table) so we can manage the tags and enable autocomplete, etc.